src/tests/client.ts

```diff
@@ -329,6 +329,51 @@ it('should not send the complete message if the socket is not open', async () =>
   await sub.waitForComplete();
 });
 
+it('should use a custom json reviver and replacer function', async () => {
+  const { url } = await startTServer({
+    replacer: (_, value) => {
+      if (typeof value == 'string') {
+        return value.toUpperCase();
+      }
+      return value;
+    },
+    reviver: (_, value) => {
+      if (typeof value == 'string') {
+        return value.toLowerCase();
+      }
+      return value;
+    },
```

Comment:
Consider dropping the `startTServer` `replacer` and `reviver` options for consistency and a client focused test.
```suggestion
```